tests/goth_tests/test_instance_restart/requestor.py

```diff
@@ -39,6 +39,9 @@ async def start(self):
 
         global instances_started
 
+        self._ctx.deploy()
+        self._ctx.start()
```

Comment:
and probably also here, though it's less important... 

```suggestion
        async for script in super().start():
            yield script
```